### PR TITLE
FEAT: support for ETH

### PR DESCRIPTION
**Pull request by rastajpa**

### CODE REVIEW

**Comment by rastajpa** [comment]
Waiting for the latest changes on CWC

**Comment by gabrielbazan7** [comment]
ACK!!

**Comment by gabrielbazan7** [comment]
ACK !

File: src/app/app.component.html [hunk 1]
```diff
@@ -62,20 +62,22 @@ <h4 class="card-title">WALLET CONFIGURATION</h4>
...
-            <span *ngIf="chain == 'btc/testnet'">testnet</span>
-            <span *ngIf="chain == 'bch/livenet'">Cash livenet</span>
-            <span *ngIf="chain == 'bch/testnet'">Cash testnet</span>
-            <span *ngIf="chain == 'bsv/livenet'">Satoshi livenet</span>
+            <label for="chain">Chain: </label>
+            <span *ngIf="chain == 'btc/livenet'">Bitcoin livenet</span>
+            <span *ngIf="chain == 'btc/testnet'">Bitcoin testnet</span>
+            <span *ngIf="chain == 'bch/livenet'">Bitcoin Cash livenet</span>
+            <span *ngIf="chain == 'bch/testnet'">Bitcoin Cash testnet</span>
+            <span *ngIf="chain == 'bsv/livenet'">Bitcoin Satoshi livenet</span>
```
[the remark this answers is not in the document]
Comment (rastajpa) [reply src/app/app.component.html]: Done!

**Pull request merged**